packages/react-components/react-avatar/src/components/AvatarGroupOverflow/useAvatarGroupOverflowStyles.ts

```diff
@@ -59,6 +59,12 @@ const useTriggerButtonStyles = makeStyles({
     },
   },
 
+  pie: {
+    backgroundColor: tokens.colorTransparentBackground,
+    ...shorthands.borderColor(tokens.colorTransparentStroke),
+    color: 'transparent',
```

Comment:
(nit) How about using opacity instead? It's more resilient to any other styles that might get added that affect the base trigger button styling.
```suggestion
    opacity: '0',
```

Reply:
@behowell This one has to stay like this since the outline of the button acts as the Pie's outline. It is important to use the button for the outline so that when focused the correct behavior happens. Another reason is that since we are using `clip-path` for the root, we cannot use a custom focus indicator with `focus-within`. If this is an issue, I can work on a follow up PR to change it.

Reply:
Ah good to know. This is fine then!